### PR TITLE
chore(website): adds ts config, upgrades deps

### PR DESCRIPTION
**Pull request by kendallstrautman**
## What this does
-  Upgrades `nextjs-scripts` and other react components
-  Swaps marketing assets, icons 
- Adjusts styles to align with new branding

### CODE REVIEW

**Comment by kendallstrautman** [comment]
TODO: upgrade package downloads page to stable

File: website/pages/_app.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default function App({ Component, pageProps }) {
         description="Packer is a free and open source tool for creating golden images for multiple
         platforms from a single source configuration."
         image="https://www.packer.io/img/og-image.png"
-        icon={[{ href: '/favicon.ico' }]}
+        icon={[{ href: '/_favicon.ico' }]}
```
Comment (mwickett) [inline website/pages/_app.js:33]: Why the underscore here?

**Review by mwickett — APPROVED** [review]
Looking good!

File: website/pages/_app.js (hunk 1 above)
Comment (kendallstrautman) [reply website/pages/_app.js:33]: Oh git wasn't picking up the change when it was the exact same filename

Comment (kendallstrautman) [reply website/pages/_app.js:33]: I can change it back now if need be

**Pull request merged**